.env.example

```
@@ -0,0 +1,158 @@
+# Node environment (development, production, local)
+# Default: "local"
+NEXT_PUBLIC_NODE_ENV="local"
+
+# CORS origin - Base URL for API requests
+# Required by: All apps
+NEXT_PUBLIC_API_URL="http://localhost:3002"
+
+# Allowed origin pattern for CORS validation
+# Can be "*", a URL, or wildcard pattern like "*.domain.com"
+# Required by: All apps
+NEXT_PUBLIC_ALLOWED_ORIGIN="*"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)